src/extension.ts

```diff
@@ -88,6 +88,9 @@ export const globalFileSystemWatcher: IFileSystemWatchers = {};
 const lsName = "Ansible Support";
 let lsOutputChannel: vscode.OutputChannel;
 
+// Expose settings
+export const extSettings = new SettingsManager();
```

Comment:
The settingsManager should not be exported here. Look at how the src/features/runner.ts file obtains the settings to match that implementation.

Reply:
Thanks @alisonlhart.  I converted some (the bare minimum) of vaullt.ts to behave more like runner and use the SettingsManager in local context, I trust that's what you're aiming for (though I didn't put all of the vault functions into the class).  A more targeted alternative here that minimiizes changes might be to just hold extSettings as a local var in vault.ts and change that directly from extension.ts.  Not pretty but keeps this changes focused on fixing vault rather than mixing in class refactoring to the PR.  Let me know what you think.

Reply:
I think this is acceptable @pgalbraith.